app/indices/plan_index.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+ThinkingSphinx::Index.define(:plan, with: :real_time) do
```

Comment:
Requires `bundle exec rake ts:rt:rebuild`

Reply:
Added in the PR description